### ISSUE TITLE
Deserializing the database migration failed: missing field `dataType`

### ISSUE THREAD

**Issue by alexvilchis** [prisma/migrate#449]
## Bug description
I just updated to beta.5. Now when I try to run migrate save on my database, an error is thrown telling me that the field dataType is missing. However, I don't have any field in my schema with that name. 

## Debug output
<img width="605" alt="Screen Shot 2020-05-13 at 1 36 31" src="https://user-images.githubusercontent.com/29064411/81779366-2e4f5380-94ba-11ea-9b5c-ba49a647a66c.png">

```
MigrateEngine:stderr May 13 01:32:45.797  INFO migration_engine: Starting migration engine RPC server git_hash="0c2ec197653b278b2978845ef958db88824cd82e" +0ms
  MigrateEngine:stderr May 13 01:32:45.805  INFO quaint::single: Starting a mysql pool with 1 connections. +7ms
  MigrateEngine:stderr {"is_panic":true,"message":"[migration-engine/connectors/sql-migration-connector/src/lib.rs:240:9] Deserializing the database migration failed.: Error(\"missing field `dataType`\", line: 0, column: 0)","backtrace":"   0: backtrace::backtrace::trace\n   1: backtrace::capture::Backtrace::new\n   2: user_facing_errors::Error::new_in_panic_hook\n   3: user_facing_errors::panic_hook::set_panic_hook::{{closure}}\n   4: std::panicking::rust_panic_with_hook\n   5: rust_begin_unwind\n   6: core::panicking::panic_fmt\n   7: core::option::expect_none_failed\n   8: <sql_migration_connector::SqlMigrationConnector as migration_connector::MigrationConnector>::deserialize_database_migration\n   9: migration_core::commands::list_migrations::convert_migration_to_list_migration_steps_output\n  10: <core::iter::adapters::ResultShunt<I,E> as core::iter::traits::iterator::Iterator>::next\n  11: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter\n  12: core::iter::adapters::process_results\n  13: <std::future::GenFuture<T> as core::future::future::Future>::poll\n  14: <std::future::GenFuture<T> as core::future::future::Future>::poll\n  15: <std::future::GenFuture<T> as core::future::future::Future>::poll\n  16: migration_core::api::rpc::RpcApi::run_command::{{closure}}\n  17: <std::future::GenFuture<T> as core::future::future::Future>::poll\n  18: <futures_util::compat::compat03as01::Compat<Fut> as futures::future::Future>::poll\n  19: <futures::future::lazy::Lazy<F,R> as futures::future::Future>::poll\n  20: futures::future::chain::Chain<A,B,C>::poll\n  21: <futures::future::then::Then<A,B,F> as futures::future::Future>::poll\n  22: <futures::future::map::Map<A,F> as futures::future::Future>::poll\n  23: <futures::future::either::Either<A,B> as futures::future::Future>::poll\n  24: futures::task_impl::std::set\n  25: migration_engine::main::{{closure}}\n  26: tokio::runtime::enter::Enter::block_on\n  27: tokio::runtime::context::enter\n  28: tokio::runtime::handle::Handle::enter\n  29: migration_engine::main\n  30: std::rt::lang_start::{{closure}}\n  31: std::panicking::try::do_call\n  32: __rust_maybe_catch_panic\n  33: std::rt::lang_start_internal\n  34: main\n"} +141ms
 ERROR  Oops, an unexpected error occured!
Error in migration engine.
Reason: [migration-engine/connectors/sql-migration-connector/src/lib.rs:240:9] Deserializing the database migration failed.: Error("missing field `dataType`", line: 0, column: 0)
```

## How to reproduce
I don't have a reproduction, but I submitted the error report #5294. Here's what I did
1. Upgrade to beta.5
2. Run migration with `npx prisma migrate save --experimental -c`
3. See error

## Expected behavior
I expected the migration to be saved in the migrations dir.

## Environment & setup
- OS: macOS
- Database: MySQL
- Prisma version: beta.5
- Node.js version: 12.16.1

**Comment by ethernal** [comment]
I submitted Error report no. 5310 with the same issue.

**Comment by Santas** [comment]
I submitted error report id 5321 with the same issue. Let me know if you need more information.

OS: macOS / Heroku
Database: Postgres 12
Prisma version: beta.5
Node: 14.1.0

**Comment by tomhoule** [comment]
The error message was improved and points to the right solution in the latest alpha (https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-migrate#troubleshooting)

**Comment by pantharshit00** [comment]
Can you all confirm that after doing the troubleshooting steps, this is fixed for you?

**Comment by tomhoule** [comment]
Thanks for taking the time to report this issue!

We've recently released a Prisma Migrate Preview ([2.1.3.0 release notes](https://github.com/prisma/prisma/releases/tag/2.13.0)), which has quite a few changes compared to the previous experimental version. We believe this issue is no longer relevant in this new version, so we are closing this.

We would encourage you to try out Prisma Migrate Preview. If you come across this or any other issue in the preview version, please feel free to open a [new issue in prisma/prisma](https://github.com/prisma/prisma/issues/new/choose).
For general feedback on Prisma Migrate Preview, feel free to chime in [on this issue](https://github.com/prisma/prisma/issues/4531).
